common/ASC.Api.Core/Core/ApiContext.cs

```diff
@@ -265,7 +265,7 @@ public static class ApiContextExtension
     {
         public static bool Check(this ApiContext context, string field)
         {
-            return context == null || context.Fields == null || (context.Fields != null && context.Fields.Contains(field));
+            return context == null || context.Fields == null || (context.Fields != null && (context.Fields.Contains(field) || context.Fields.Contains(field.ToLower())));

```

Comment:
You can use the Contains method with StringComparer.InvariantCultureIgnoreCase parameter